core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -1828,7 +1830,7 @@ void skip0() {
     /**
      * Represents a process tree over a channel.
      */
-    public static class Remote extends ProcessTree implements Serializable {
+    public static class Remote extends ProcessTree {
```

Comment:
Could it be a breaking change? I know it comes from https://github.com/jenkinsci/jenkins/pull/3144 , but I am not sure why I did that

Reply:
This `implements Serializable` clause was always redundant, given the superclass:

```java
public abstract class ProcessTree implements Iterable<OSProcess>, IProcessTree, Serializable
```

Reply:
Right. One month without coding for me :D